src/mixing.html

```diff
@@ -1,31 +1,112 @@
 <section data-max-toc="3">
- <h2 id="mixing">Mixing Markup Languages for Mathematical Expressions</h2>
+ <h2 id="mixing">Semantic Annotations</h2>
  <div class="issue" data-number="281"></div>
+ <div class="issue" data-number="252"></div>
  
- <p>MathML markup can be combined with other markup languages, and
- these mixing constructions are realized by the semantic annotation
- elements.  The semantic annotation elements provide an important tool
- for making associations between alternate representations of an
+ <p>Semantic annotation refers to markup added to
+ make associations between alternate representations of an
  expression, and for associating semantic properties and other
- attributions with a MathML expression.  These elements allow
- presentation markup and content markup to be combined in several
- different ways.  One method, known as <em>mixed markup</em>, is to
- intersperse content and presentation elements in what is essentially a
- single tree.  Another method, known as <em>parallel markup</em>,
- is to provide both explicit presentation markup and content markup in
- a pair of markup expressions, combined by a single <code class="element">semantics</code>
- element.
+ attributions with a MathML expression.</p>
+ <p>The most general mechanism provided by MathML is the <code
+ class="element">semantics</code> element. This allows presentation
+ markup and content markup to be combined in several different ways
+ and for MathML to embed markup from other vocabularies such as
+ [[OpenMath2004]] or [[SVG]]. This general mechanism is described
+ below in <a href="#mixing_semantic_annotations"></a> but first we describe a new
+ feature added in MathML&#160;4 to faciliate annotating Presentation
+ MathML with the <q>intent</q> or semantic meaning of the term.
  </p>
 
+ <section>
+  <h3 id="mixing_intent">Semantic Intent for Presentation MathML</h3>
```

Comment:
Is it worth pondering, for a brief moment, that `intent` at least as we've defined it so far, could apply equally well to content, and my conceivably be useful. (though of course defaulting patterns would be different)

Reply:
Note on "semantic intent"

A big part of the discussion that led to the `intent` term, for the attribute name, was to very carefully avoid the baggage (and associations) which the word "semantic" brought along. Especially given the spec has both a Content MathML dialect, *and* a `<semantics>` element.

So far I have been using the phrase "author intent" when "intent" needed to be qualified, as well as "encyclopedic intent" when discussing the way in which I have been choosing the values in the lists of sample values.

But I would be equally happy with "accessible intent" or the simple and neutral "intent attribute", in such a heading title followed by "for Presentation MathML".

Just trying to keep the reader's mind free of prior bias, when first encountering the new attribute.

Reply:
I had a similar thought while discussing wih @samdooley  on last week's call the strict content rewrite rules. I'll open an issue on that and add an issue link to this section.

Reply:
There was some discussion of this in #289 with a decision to park this at least for this MathML4 iteration, so marking this as resolved.